Model/PurgeCache.php

```diff
@@ -59,10 +60,10 @@ public function sendPurgeRequest($tagsPattern)
             'applicationId' => $application_id,
             'environmentName' => $environment_name,
             'proxyName' => $proxy_name,
-            'uriStem'   => '/state?banExpression=' . urlencode('obj.http.X-Magento-Tags ~ ' . $tagsPattern)
+            'uriStem'   => '/state?banExpression=' . urlencode('obj.http.X-Magento-Tags ~ ' . $tagsPattern) . '&async=true'
```

Comment:
Re-order `/state?async=true&banExpression=`. Always put what we control ahead of what we don't control.